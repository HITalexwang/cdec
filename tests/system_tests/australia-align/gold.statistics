-lm_nodes 77
-lm_edges 244232
-lm_paths 3.79555e+28
-lm_trans australia is have diplomatic relations with north korea one of the few countries .
-lm_viterbi -12.7893
constr_nodes 111
constr_edges 305
constr_paths 9899
