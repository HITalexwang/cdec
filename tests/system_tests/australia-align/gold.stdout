0-0 1-1 2-11 3-12 3-13 4-7 4-8 5-9 5-10 6-4 7-5 8-6 9-2 9-3 10-14
